docs/tutorials/advanced-guides/how-to-login-to-toolkit-using-ledger.md

```diff
@@ -0,0 +1,124 @@
+
+If you don't have the required amount yet then you can utilize the `Buy UOS` option of the Ultra Wallet extension.
+
+![](../guides/images/UltraProWallet/wallet-buy-uos.png)
+
+When you have enough funds you can proceed with the creation of the Ultra Pro Wallet. For this you need to click on the profile icon and select the `Create an Ultra Pro Wallet` button.
+
+![](../guides/images/UltraProWallet/wallet-create-pro-wallet.png)
+
+This will open a new browser tab where you will have 2 options:
+
+-   Generate a new private and public key pair
+-   Use an existing pair to create an Ultra Pro Wallet
+
+![](../guides/images/UltraProWallet/wallet-generate-or-paste-public-key.png)
+
+Since we already have the public key associated with our Ledger, we will simply paste the public key in the input.
+
+![](./images/ultra-wallet-create-ultra-pro-wallet.png)
+
+Before finishing the Ultra Pro Wallet creation you need to specify the maximum amount of UOS your are willing to pay to create it. The price fluctuates over time due to USD to UOS conversion fluctuations and may be a bit different then the estimate shown on the page. Pick a value that is a bit over the estimated price or just put a large number there, we will charge only the current equivalent of 2 USD converted to UOS.
+
+Now you can press the `Create Account` button to proceed. You will be prompted to approve the transaction with its details listed. Check it and click `Confirm`
+
+![](../guides/images/UltraProWallet/wallet-approve-transaction.png)
+
+If everything was done correctly you will get a successful transaction screen. You can click on `Block Explorer` button to view the transaction status.
+
+![](../guides/images/UltraProWallet/wallet-transaction-successful.png)
+
+Your new Ultra Pro Wallet name will be listed in the actions list. This Ultra Pro Wallet name is now associated with your Ledger device.
+
+![](../guides/images/UltraProWallet/wallet-check-block-explorer.png)
+
+## Login to Toolkit
+
+Return to Ultra Toolkit homepage (https://toolkit.ultra.io), and click on the network selection component on the top right of your screen.
+
+![](../fundamentals/images/toolkit-network-selection.png)
+
+Clicking on the network selection component will open up a list of available networks that you can use the toolkit on. From the list of available networks, click on `Mainnet`.
+
+Click on the `Login To Toolkit` button to open a list of all supported login methods, and then select the `Ledger` option.
+
+![](../fundamentals/images/toolkit-login-button.png)
+
+![](./images/toolkit-login-options-ledger.png)
+
+The new prompt will ask you to input your Ledger index. Enter the index number that you selected earlier and click `Select` button. It will load your Ultra Pro Wallet account associated with that specific index.
+
+![](./images/toolkit-ledger-login-index.png)
+
+![](./images/toolkit-ledger-account-selection.png)
```

Comment:
Can you please for this screenshot edit the page with inspect element to use the usual account name `1aa2aa3aa4in` so it matches the other screenshots?